nrich-search/README.md

```diff
@@ -38,32 +40,50 @@ public class ApplicationConfiguration {
     public RepositoryFactorySupportFactory searchRepositoryFactorySupportFactory(StringToEntityPropertyMapConverter stringToEntityPropertyMapConverter) {
         return new SearchRepositoryFactorySupportFactory(stringToEntityPropertyMapConverter);
     }
-}
 
+}
 ```
 
-`StringToTypeConverter<Object>` performs conversion from string to typed instances and is used when querying registry entities. Default implementation (`DefaultStringToTypeConverter`)
-accepts a list of data formats and regular expressions that are used to convert string to type found in properties of entity classes.
+### [StringToTypeConverter][string-to-type-converter-url]
+
+[`StringToTypeConverter<Object>`][string-to-type-converter-url] performs conversion from string to typed instances and is used when querying registry entities.
```

Comment:
Maybe just `StringToTypeConverter` (without `<Object>`). I think we have this in multiple places.

Additionally, since we now have a link to class here, maybe remove it from the heading?  In this case, I prefer a link inside the text.
We have this in multiple places.

Reply:
I agree with you, Ill change it everywhere where its applicable.